workflows/phylorthology.nf

```diff
@@ -7,38 +7,60 @@
 def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 
 // Validate input parameters
-WorkflowPhylorthology.initialise(params, log)
+//WorkflowPhylorthology.initialise(params, log)
 
 // TODO nf-core: Add all file path parameters for the pipeline to the list below
 // Check input path parameters to see if they exist
-def checkPathParamList = [ params.input, params.multiqc_config, params.fasta ]
-for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
+//def checkPathParamList = [ params.input, params.fasta ]
+//for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
 
 // Check mandatory parameters
-if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
+if (params.input) { ch_input = file(params.input) } else { exit 1, 'Full samplesheet not specified!' }
```

Comment:
I believe I've described in another comment, but originally these held the different datasets (complete and the reduced one for MCL testing), and these channels were used when preparing for orthofinder (i.e. so that orthofinder would make all of its bespoke directory structures at these locations). 

In a revised version to handle data on s3 (will soon be pushed as a new branch) these are no longer needed/used, since the the prep will all be done in the working directory for that process.